app/models/pano/pano_form_builder.rb

```diff
@@ -2,7 +2,8 @@ module Pano
   class PanoFormBuilder < ActionView::Helpers::FormBuilder
     def formatted_date_field(name, options = {})
       hidden_field = @template.hidden_field(@object_name, name, data: {target: 'hidden'})
-      date = @template.tag('date-input', {name: "#{@object_name}[#{name}]", role: "textbox", value: @object.read_attribute(name) || @object.send(name), id: sanitize_to_id("#{@object_name}[#{name}]"), tabindex: 0, active: false}.update(options.stringify_keys))
+      # Format value to be parseable by new Date() in IE11. Use to_time not to_date, to prevent date misalignment
+      date = @template.tag('date-input', {name: "#{@object_name}[#{name}]", role: "textbox", value: @object.read_attribute(name) ? @object.read_attribute(name).to_time.iso8601 : (@object.send(name) ? @object.send(name).to_time.iso8601 : @object.send(name)), id: sanitize_to_id("#{@object_name}[#{name}]"), tabindex: 0, active: false}.update(options.stringify_keys))
```

Comment:
`@object.read_attribute(name) ? @object.read_attribute(name).to_time.iso8601 : (@object.send(name) ? @object.send(name).to_time.iso8601 : @object.send(name)), id: sanitize_to_id("#{@object_name}[#{name}]")`

@suzukia could you convert this ternary to a method to make it more readable? Maybe also include some comments around what the cases are.